types/plugins.d.ts

```diff
@@ -0,0 +1,38 @@
+export interface SanityBuildPluginEntity {
```

Comment:
This file is in #820, but there is also a small update in here. Once #820 is merged, the changes here should be a one liner.